src/lib/structures/Command.ts

```diff
@@ -69,13 +60,24 @@ export abstract class Command extends BaseAliasPiece {
 		this.description = options.description ?? '';
 		this.preconditions = new PreconditionContainerAll(this.client, options.preconditions ?? []);
 		this.extendedHelp = options.extendedHelp!;
-		this.guarded = options.guarded!;
-		this.hidden = options.hidden!;
 		this.flags = options.flags!;
-		this.quotedStringSupport = options.quotedStringSupport!;
+		this.#lexer.setQuotes(
+			options.quotes ?? [
```

Comment:
This should probably ensure quotes is an array

Reply:
Why? We don't for any of the other keys.